sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/SyncSample.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.digitaltwins.core;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.digitaltwins.core.models.DigitalTwinsGetByIdResponse;
+import com.azure.identity.ClientSecretCredentialBuilder;
+
+public class SyncSample
+{
+    public static void main(String[] args)
+    {
+        String tenantId = System.getenv("TENANT_ID");
+        String clientId = System.getenv("CLIENT_ID");
+        String clientSecret = System.getenv("CLIENT_SECRET");
+        String endpoint = System.getenv("DIGITAL_TWINS_ENDPOINT");
+        String digitalTwinId = System.getenv("DIGITAL_TWIN_ID");
+
+        TokenCredential tokenCredential = new ClientSecretCredentialBuilder()
+            .tenantId(tenantId)
+            .clientId(clientId)
+            .clientSecret(clientSecret)
+            .build();
+
+        DigitalTwinsClient client = new DigitalTwinsClientBuilder()
+            .tokenCredential(tokenCredential)
+            .endpoint(endpoint)
+            .buildClient();
+
+        DigitalTwinsGetByIdResponse syncResponse = client.getDigitalTwin(digitalTwinId);
+        Object digitalTwin = syncResponse.getValue();
```

Comment:
As an interesting aside, the actual type of the returned digital twin here is a map. Not a Json object, strictly speaking.

Reply:
yeah, I remember that being an issue in the previous java autorest as well, it didn't like json strings for some reason; always wanted to return it as a linkedmap or something similar